### DISCUSSION TITLE
FYI There is an unofficial fork with credits removed which has published a website

### DISCUSSION THREAD

**Discussion by sylikc** [sylikc/jpegview#31] [General]
Just FYI, in case my comment gets deleted over at the new repository, I just noticed an unofficial fork (well none of this is really official) but it removed all the documentation and history of the original repository (this repo maintains all of David Kleiner's original history)

It's also "published" with a new version number 1.1.1.0, even though it probably matches the commit from 1.0.37.1 here.

```
https://
github.com/
annh9b/JPEGView-Image-Viewer-and-Editor
```

It adds an "official" site, but given that the author has removed any credits to past contributors, I'd be wary of the project (I can't vouch for the release)
```
https://
www
.open-source
.tech/
JPEGView-Image-Viewer-and-Editor/
```

All just as an FYI.  The "official" which is the vetted releases here are released on GitHub only at this time

(I don't include direct link in case some search engine starts ranking it higher.)

**Comment by pureby** [comment]
Thanks for the heads-up!

I've checked that Github page and the developer has since included a reference to the original project, at least in Readme file. He also mentions incorporating updates from _this_ — @sylikc's — fork, and some weird "collaboration with Facebook"... (?!)

It doesn't look like the developer has any malicious intent (at least not at this point in time), but it does seem like a hobby project for private use made by someone who has no idea what he is doing.

Either way, best to stay away 👎
